docs/ADL2/master01-preface.adoc

```diff
@@ -65,6 +65,26 @@ For existing users of ADL or archetype development tools, the following provides
 
 The object syntax used to represent the description, terminology and annotations sections of an ADL archetype has been historically known as 'dADL' (i.e. 'data ADL'). Since this syntax is completely generic and has no specific dependency on either ADL or openEHR, it has been separated out into its own specification known as Object Data Instance Notation (ODIN).
 
+=== ADL 2.4
+
+==== Changes
+
+ADL2 was initially released with a new id-coded coding system using id-codes, at-codes and ac-codes.
+
+The primary change in version 2.4 is the introduction of an alternative at-code coding system, identical to that used in ADL1.
+
+Although a conversion algorithm was developed to enable conversion for ADL1 at-codes in archetypes and templates to the new id-codes and coding system, the openEHR implementer community had concerns on the burden and impact and safety risks inherent in converting downstream software artefacts, queries and persisted patient records.
+
+It was decided that the best solution was to allow openEHR-RM based systems and archetypes to continue to use the original ADL1 at-coded coding system instead of the id-coded coding system. This was seen as critical to remove a very significant barrier to transition from ADL1 to ADL2, which in all other respects was widely welcomed by the established openEHR implementer community.
+
+Further information on the choice of coding system is provided at <<_node_identifier_and_coding_systems>>.
+
+NOTE: ADL tools conformant to ADL/AOM 2.3 or earlier will not conform to the at-coded ADL2 archetypes described in ADL 2.4 without additional engineering.
+
+==== Backward Compatibility
+
+When using the at-coded coding system, this ADL2.4 release is fully backwards compatible with openEHR data created based on ADL 1.4 archetypes but breaks compatibility with openEHR data created based on ADL 2.0 - ADL 2.3 archetypes.
```

Comment:
```suggestion
When using the AT-coded coding system, this ADL2.4 release is fully backwards compatible with openEHR RM data created based on ADL 1.4 archetypes. But it breaks compatibility with openEHR RM data created using an ID-coded coding system, which was the only option in ADL 2.0 - ADL 2.3 archetypes. So ID-coded ADL2 systems will need to migrate their openEHR RM data in order to be compatible with at-coded openEHR RM data.
```

Reply:
I think it's good enough. But to be more precise, it's about the archetypes, not the data persé, you can't have the same archetype between at and id coded systems, so you can't have the benefits that brings.

Reply:
We should also mention origin 0 in ID codes vs origin 1 in at codes right? Because it can create clinical safety issues if you mix them.